src/assets/translations/en/main.json

```diff
@@ -2635,5 +2636,12 @@
         "subtitle": "Discover recent launches."
       }
     }
+  },
+  "foxPage": {
+    "title": "FOX Token Dashboard",
+    "description": "Since 2019, our shapeshifting FOX Token has been offering users an ever-expanding world of utility and advantages. Today, our ERC-20 governance token gives you fee discounts, a share in revenues on Thorchain, and enables you to influence the future of ShapeShift through your vote. Stake, swap, buy, supply, and manage your FOX holdings here.",
```

Comment:
```suggestion
    "description": "Since 2019, our shapeshifting FOX Token has been offering users an ever-expanding world of utility and advantages. Today, our ERC-20 governance token gives you fee discounts, a share in revenues on THORChain, and enables you to influence the future of ShapeShift through your vote. Stake, swap, buy, supply, and manage your FOX holdings here.",
```